packages/scene-composer/src/components/panels/scene-settings/FogSettingsEditor.spec.tsx

```diff
@@ -0,0 +1,147 @@
+import wrapper from '@awsui/components-react/test-utils/dom';
+import { render } from '@testing-library/react';
+import React from 'react';
+
+import { useStore } from '../../../store';
+import { KnownSceneProperty } from '../../../interfaces';
+
+import { FogSettingsEditor } from './FogSettingsEditor';
+
+jest.mock('@awsui/components-react', () => ({
+  ...jest.requireActual('@awsui/components-react'),
```

Comment:
We want to avoid use of `requireActual` as it doesn't use mocked data. 

reference: https://jestjs.io/docs/jest-object#jestrequireactualmodulename

Reply:
Removing this makes polaris wrapper tests fail unfortunately.

Reply:
Since it's an existing pattern we can let this go in. I'll try to carve out some time to fix this implementation across the board later.